train_supervised.py

```diff
@@ -47,36 +49,35 @@
 testloader = torch.utils.data.DataLoader(testset, batch_size=test_batch_size,
                                          shuffle=False, num_workers=2)
 
-#define the device
+# define the device
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
 # define the model
-#model = wide_resnet.Wide_ResNet(depth=depth, widen_factor=widen_factor, dropout_rate=dropout_rate, num_classes=num_classes)
+# model = wide_resnet.Wide_ResNet(depth=depth, widen_factor=widen_factor, dropout_rate=dropout_rate, num_classes=num_classes)
 model = resnet_official.wrn_28_2().to(device)
 
-#define the optimizer and criterion
+# define the optimizer and criterion
 criterion = nn.CrossEntropyLoss()
-optimizer = optim.Adam(model.parameters(), lr=learning_rate, betas=[.9, .999])
-
+optimizer = optim.Adam(model.parameters(), lr=learning_rate, betas=(.9, .999))
 
 print("starting training ...")
 
 experiment = replicate.init(
-        path=".",
-        params={"learning_rate": params["learning_rate"], "num_epochs": params["num_epochs"]},
-    )
+    path=".",
+    params={"learning_rate": params["learning_rate"], "num_epochs": params["num_epochs"]},
+)
 
-#create a real training loop
+# create a real training loop
 for epoch in range(num_epochs):
-    for i, data in tqdm(enumerate(trainloader), total=len(trainset) // train_batch_size + 1):
+    for i, (inputs, labels) in tqdm(enumerate(trainloader), total=len(trainset) // train_batch_size + 1):
 
-        #obtain data
-        inputs, labels = data[0].to(device), data[1].to(device)
+        # obtain data
+        inputs, labels = inputs.to(device), labels.to(device)
 
```

Comment:
You can unpack the `data` tuple right in the for loop, which is more "Pythonic".